createdisk.sh

```diff
@@ -104,25 +104,24 @@ EOF
 
 # Add gvisor-tap-vsock service
 ${SSH} core@${VM_IP} 'sudo bash -x -s' <<EOF
-  tee /etc/systemd/system/gv-user-network@.service <<TEE
+  tee /etc/systemd/system/gvisor-tap-vsock.service <<TEE
```

Comment:
Is the change from a template unit to a non-template one required? Imo the `BindsTo`/`After`/`ExecStartPre` changes would be enough to **workaround** the issue you are seeing.

Reply:
@cfergeau Since we create tap0 (which is hardcoded) using nmcli does it matter to use template unit vs non-template one?

Reply:
just removing the `BindsTo` is enough.. 

but the downside is that, now i think if the `tap0` device disappears and reappears, the service will not be re-started